tests/test_to_number.cc

```diff
@@ -45,6 +47,25 @@ struct TestConfig {
     unsigned static const number_of_random_tests{ 5'000 };
 };
 
+template <typename T>
+inline int error_in_ulp(T value, T reference)
```

Comment:
```suggestion
int error_in_ulp(T value, T reference)
```

Reply:
Tests?

Reply:
Waaah !!!

I did implement a test and **found a bug** :grimacing: